cicd/pipelines/working-with-pipelines-web-console.adoc

```diff
@@ -0,0 +1,49 @@
+:_content-type: ASSEMBLY
+[id="working-with-pipelines-web-console"]
+= Working with {pipelines-title} in the web console
+include::_attributes/common-attributes.adoc[]
+:context: working-with-pipelines-web-console
+
+toc::[]
+
+You can use the *Administrator* or *Developer* perspectives to create and modify `Pipeline`, `PipelineRun`, and `Repository` objects from the *Pipelines* page in the {product-title} web console.
```

Comment:
It seems to me that it's either "use the Administrator and Developer perspectives" or  "use the Administrator or Developer perspective"